lib/stripe/checkout/session.ex

```diff
@@ -14,12 +14,42 @@ defmodule Stripe.Session do
   import Stripe.Request
 
   @type line_item :: %{
-          :amount => integer(),
+          optional(:name) => String.t(),
+          optional(:quantity) => integer(),
+          optional(:adjustable_quantity) => adjustable_quantity(),
+          optional(:amount) => integer(),
+          optional(:currency) => String.t(),
+          optional(:description) => String.t(),
+          optional(:price) => String.t(),
+          optional(:price_data) => price_data,
+          optional(:images) => list(String.t())
```

Comment:
Should we also add 

`dynamic_tax_rates`
`tax_rates`

Reply:
Oh, I hadn't noticed those. I was focused on the `line_items`.

I'll add those as well.

- [ ] dynamic tax rates
- [ ] tax rates

Reply:
I updated the typespec with these fields.

As far as I can tell they are both just a list of tax rate IDs.

[Tax Rate Checkout Docs](https://stripe.com/docs/payments/checkout/taxes#fixed-tax-rates)
[Dynamic Tax Rate Checkout Docs](https://stripe.com/docs/payments/checkout/taxes#dynamic-tax-rates)